src/Api/Utilities/ExceptionHandlerFilterAttribute.cs

```diff
@@ -54,18 +54,17 @@ public override void OnException(ExceptionContext context)
                     errorMessage = badRequestException.Message;
                 }
             }
-            else if (exception is StripeException stripeException &&
-                stripeException?.StripeError?.ErrorType == "card_error")
+            else if (exception is StripeException stripeException && stripeException?.StripeError?.Type == "card_error")
```

Comment:
`stripeException` wouldn't be null in the `is` statement in the order of operations for evaluating this `if` condition, so the first `?` wouldn't be necessary in `stripeException?.StripeError?.Type`